src/NetworkEquipmentModelStencil.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2023 Teclib' and contributors.
+ * @copyright 2003-2014 by the INDEPNET Development Team.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+use Glpi\Application\View\TemplateRenderer;
+
+class NetworkEquipmentModelStencil extends Stencil
+{
+    public static function getTypeName($nb = 0): string
+    {
+        return __('Graphical slot definition');
+    }
+
+    public static function displayTabContentForItem(CommonGLPI $item, $tabnum = 1, $withtemplate = 0)
+    {
+        self::displayStencilEditor($item);
+
+        return true;
+    }
+
+    public static function displayStencil(CommonDBTM $item, array $picture_fields = [], array $params = [])
```

Comment:
IMHO, all display methods could be moved into an interface (with a corresponding trait, if possible), and it would be easier to use.
Also, please avoid using static methods.

```php
namespace Glpi\Features;

interface ZonableModelPicture
{
    public function displayStencil(): void;

    public function displayStencilEditor(): void;
}
```

To use it in PHP:
```php
// to display the stencil
if ($item instanceof ZonableModelPicture) {
    $item->displayStencil();
}

// to display editor
if ($item instanceof ZonableModelPicture) {
    $item->displayStencilEditor();
}
```